train_configs/llama3_8b.toml

```diff
@@ -37,6 +37,9 @@ tensor_parallel_degree = 1
 compile = false
 dataset = "c4"
 
+[data_parallel]
+reshard_after_forward_policy = "default" # default / never / always
```

Comment:
nit: not sure if its preferred to add a new section. I actually don't hate it, but then its a bit divergent from the existing args in the [training] section.  Should we just put this into training heading for now and discuss changing the structure later?

(e.g. `training.data_parallel_reshard_after_forward`)  yea, its a bit long/wordy.  but maybe ok.  I dropped 'policy' since the argparse 'choices' will forcibly make the options clear

Reply:
Yeah good point. I switched this to e.g. `training.fsdp_reshard_after_forward` for now - took the liberty of swapping `data_parallel` to `fsdp` so it's more explicit that this only applies to fsdp.